scan/lib/scan/test_command_generator.rb

```diff
@@ -37,6 +37,7 @@ def options
         options << "-resultBundlePath '#{result_bundle_path}'" if config[:result_bundle]
         options << "-enableCodeCoverage YES" if config[:code_coverage]
         options << "-enableAddressSanitizer YES" if config[:address_sanitizer]
```

Comment:
Good point! Do you think same should apply to thread sanitizer and code coverage options too?